pkg/virt-api/webhooks/mutating-webhook/mutators/vm-mutator_test.go

```diff
@@ -220,11 +232,15 @@ var _ = Describe("VirtualMachine Mutator", func() {
 
 		vm.Spec.Template.Spec.Domain.Machine = &v1.Machine{Type: "pc-q35-2.0"}
 
-		vmSpec, _ := getVMSpecMetaFromResponseCreate(rt.GOARCH)
+		vmSpec, _ := getVMSpecMetaFromResponseCreateWithArch(arch)
 		Expect(vmSpec.Template.Spec.Domain.Machine.Type).To(Equal(vm.Spec.Template.Spec.Domain.Machine.Type))
-	})
+	},
+		Entry("amd64", "amd64"),
```

Comment:
What about `ppc64le`? Can we remove remnants of it in a follow up? I don't think it is supported by KubeVirt anymore?